blocks/inspector-controls/base-control/index.js

```diff
@@ -8,11 +8,12 @@ import classnames from 'classnames';
  */
 import './style.scss';
 
-function BaseControl( { id, label, className, children } ) {
+function BaseControl( { id, label, help, className, children } ) {
 	return (
 		<div className={ classnames( 'blocks-base-control', className ) }>
 			{ label && <label className="blocks-base-control__label" htmlFor={ id }>{ label }</label> }
 			{ children }
+			{ !! help && <p className="blocks-base-control__help">{ help }</p> }
```

Comment:
Do we need any accessibility semantics here? `aria-describedby`?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

Do we need this to be built into the controls themselves, or is it fine if they're just the next sibling in the DOM (i.e. `<TextControl /><p>Help Text</p>`). Otherwise, we ought to be consistent in how all of the controls accept a `help` prop.